File: backend/Algortimo/Models/Cardapio.cs
namespace Algoritimos.Models
{
    public class Cardapio
    {
        public int NumeroPratos { get; set; }
        public int NumeroDias { get; set; }
        public int Orcamento { get; set; }
        public PratosInfor PratosInformacoes { get; set; }
    }
}

File: restaurante-orc/src/type/Cardapio.ts


export type Cardapio ={
  numeroPratos: number
  numeroDias: number
  orcamento: number
  pratosInformacoes: PratosInformacoes
}

export type PratosInformacoes ={
  pratos: Prato[]
}

export type Prato ={
  id: number
  custo: number
  lucro: number
}
